hudi-cli/src/main/java/org/apache/hudi/cli/utils/SparkTempViewProvider.java

```diff
@@ -101,6 +101,17 @@ public void runQuery(String sqlText) {
     }
   }
 
+  @Override
+  public void showAllViews() {
+    try {
+      sqlContext.sql("SHOW TABLES").show(Integer.MAX_VALUE, false);
+    } catch (Throwable ex) {
+      // log full stack trace and rethrow. Without this its difficult to debug failures, if any
+      LOG.error("unable to initialize spark context ", ex);
```

Comment:
IMO, this error description is not correct, right? The throwable not only covers the initialization of the spark context

Reply:
> IMO, this error description is not correct, right? The throwable not only covers the initialization of the spark context

Yes, change it to `unable to get all views`.